devito/ir/equations/equation.py

```diff
@@ -227,6 +263,26 @@ def func(self, *args, **kwargs):
     __reduce_ex__ = Pickable.__reduce_ex__
 
 
+class ClusterizedBreak(ClusterizedEq):
```

Comment:
yeah, basically you're doubling the IR, which is something that should be avoided at all costs